plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -462,20 +468,20 @@ load_config(plugin_state_t *pstate, invalidate_t **ilist)
             }
             if (i) {
               iptr->next = i;
-              TSDebug(PLUGIN_NAME, "Loaded %s %d %d %s", i->regex_text, (int)i->epoch, (int)i->expiry, strForResult(i->new_result));
+              TSDebug(PLUGIN_NAME, "Loaded %s %jd %jd %s", i->regex_text, i->epoch, i->expiry, strForResult(i->new_result));
```

Comment:
What guarantees that the times will be of type intmax_t?  You could change to `intmax_t(i->epoch)` to be sure.